.github/workflows/release.yml

```diff
@@ -152,7 +152,7 @@ jobs:
           title: 'chore: update versions'
           publish: pnpm release:publish
```

Comment:
although this will not publish a github release, but it will attempt to push the tags which was pushed by the npm-release step 

better to pass `--no-git-tags` to the pnpm release:publish